app.rb

```diff
@@ -1,8 +1,32 @@
 require 'sinatra/base'
+require_relative 'lib/turn'
+require_relative 'lib/computer'
 class RockPaperScissors < Sinatra::Base
+  enable :sessions
+
```

Comment:
how do your sessions work without setting a secret session to store your inputs in?

Reply:
It saves them as cookies. So it works on this, but for more extensive projects, I would have to delete them before inputting new ones... it explains it here: https://webapps-for-beginners.rubymonstas.org/sessions/sinatra_sessions.html , so probably not the best practice!